pkg/services/annotations/annotations.go

```diff
@@ -98,13 +98,13 @@ type ItemDTO struct {
 	AlertId      int64            `json:"alertId"`
 	AlertName    string           `json:"alertName"`
 	DashboardId  int64            `json:"dashboardId"`
-	DashboardUid string           `json:"dashboardUid"`
 	PanelId      int64            `json:"panelId"`
 	UserId       int64            `json:"userId"`
 	NewState     string           `json:"newState"`
 	PrevState    string           `json:"prevState"`
 	Created      int64            `json:"created"`
 	Updated      int64            `json:"updated"`
+	DashboardUid string           `json:"dashboardUid"`
```

Comment:
Nitpick: the [relocation of the `dashboard.uid` field in the `sqlstore` submodule](https://github.com/grafana/grafana/pull/31161/commits/b4ea0c7a341d2e7df3d3ba2d554bdf4626c26f9b#diff-5a382284915b31e0cfcc606446bcd3e38ad97c26fce35af6081bbdbc8c8c8d0aR139) looks good, although in this struct perhaps it makes sense to keep `DashboardUid` on the line after `DashboardId` (again, some kind of grouping-by-convention).

Reply:
Yeah, i put that there in the first commit, but i did not know if the ItemDTO must have the same order of the query. I saw that's not necessary, thanks for that point ;)